training/training_code/train-max-model.sh

```diff
@@ -13,6 +13,7 @@ POST_PROCESSING_FAILED=2
 PACKAGING_FAILED_RETURN_CODE=3
 CUSTOMIZATION_ERROR_RETURN_CODE=4
 ENV_ERROR_RETURN_CODE=5
+DATA_DIR=${DATA_DIR}/data
```

Comment:
As implemented, this change will invalidate the check in https://github.com/IBM/MAX-Question-Answering/blob/7e3bfd4a6d7b98d1a5f6980709486e43bee12ec3/training/training_code/train-max-model.sh#L24 because the `DATA_DIR` environment variable is now always defined.

See 
following https://github.com/IBM/MAX-Text-Sentiment-Classifier/blob/master/training/training_code/train-max-model.sh#L39-L45 for a solution that shouldn't have any side effects.